src/clarity/clarityValue.ts

```diff
@@ -45,6 +48,43 @@ export type ClarityValue =
   | ListCV
   | TupleCV;
 
+export function cvToString(val: ClarityValue, encoding: 'tryAscii' | 'hex' = 'tryAscii'): string {
+  switch (val.type) {
+    case ClarityType.BoolTrue:
+      return 'true';
+    case ClarityType.BoolFalse:
+      return 'false';
+    case ClarityType.Int:
+      return val.value.fromTwos(CLARITY_INT_SIZE).toString();
+    case ClarityType.UInt:
+      return val.value.toString();
+    case ClarityType.Buffer:
+      if (encoding === 'tryAscii') {
+        try {
+          return `"${val.buffer.toString('ascii')}"`;
```

Comment:
Can this use the same regex used by Clarity rust lexer to ensure the string is printable ascii?